documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,38 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application and reflects the new changes.
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+![Oauth client creation](../images/oauthClientCreation.png)
+
+Select the client tab, which provides an option to register a new client as shown below:
+
+![Register new client](../images/registerNewOauthClient.png)
+
+During registering the client, in the subscription tab, select the portal landscape you would like to subscribe to, ex: `portal/nwc` as shown below:
+
+![Portal subscription](../images/portalSubscription.png)
```

Comment:
This image shows part of the left hand panel, which should be shown earlier. Here is no need for that IMO.

Reply:
But, I do not see any issue in showing it here as well, Moreover, it gives clear picture IMO.

Reply:
I believe it adds unnecessary information. Also part of the icons for Repositories, Integration Tokens and Usage Analytics are visible. It is not a cleanly cut image, which does not look professional.